src/Laravel/Cashier/Invoice.php

```diff
@@ -171,7 +171,7 @@ public function discount()
     {
         setlocale(LC_MONETARY, $this->billable->getCurrencyLocale());
 
-        return round(money_format('%i', ($this->subtotal / 100) - ($this->total / 100)), 2);
+        return round(money_format('%!i', ($this->subtotal / 100) - ($this->total / 100)), 2);
```

Comment:
So this change was to fix an error that was occurring on the unit tests. The original method the `money_format` call was returning `USD100.00`. Then when passed into round, round was returning `0`; thus breaking the unit test as `0 !== 100`.

Adding the bang keeps money_format from returning `USD` in the string.